### ISSUE TITLE
Blade doesn't parse the @laravelPWA directive

### ISSUE THREAD

**Issue by makhosi6** [silviolleite/laravel-pwa#58]
Please I'm trying to use this package on my laravel 5 site. but Blade doesn't parse the @laravelPWA directive, instead it appears on the front-end between text tags like this, `<text>@laravelPWA</text>`
The directive is inside the `head`

**Comment by samydens** [comment]
I had the same problem. I fixed it by moving `@laravelPWA` to the bottom of the head.

**Comment by Silidrone** [comment]
@samydens solution worked

**Comment by makhosi6** [comment]
No, the solution did not work for me, and It's also failing to `Publish` (it doesn't create `config/laravelpwa.php` automatically). But when I clone the DEMO everything works.

**Comment by Silidrone** [comment]
Make sure you did `composer install` (considering it is already composer required, if not you need to: composer require `silviolleite/laravelpwa --prefer-dist`) and also make sure you ran `php artisan vendor:publish --provider="LaravelPWA\Providers\LaravelPWAServiceProvider"`

**Comment by ilikewebsite** [comment]
Blade doesn't parse the @laravelPWA directive

**Comment by ilikewebsite** [comment]
<!doctype html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Laravel - @yield('title')</title>
    
    <link href="https://fonts.googleapis.com/css?family=Nunito:200,600" rel="stylesheet">
    
    <link rel="stylesheet" href="assets/css/style.css">
    @laravelPWA
    @yield('styles')
    @livewireStyles

</head>

<body>
@yield('content')



<script src="assets/js/lib/bootstrap.bundle.min.js"></script>

<script type="module" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.js"></script>

<script src="assets/js/plugins/splide/splide.min.js"></script>

<script src="assets/js/base.js"></script>

@yield('scripts')
@livewireScripts
</body>

</html>

@laravelPWA  not working. ! Any solution ?

**Comment by IlyasDiker** [comment]
I don't know what was wrong exactly just cleared the space around added empty lines (above, bellow) the `@laravelPWA`
```php

@laravelPWA

```
and it worked, I think the library could be improved using the [https://laravel.com/docs/9.x/blade#manually-registering-components](Manually registered component) and end up using something similar to this `<x-laravelpwa::meta/>` instead of the directive.

**Comment by QazCetelic** [comment]
![image](https://user-images.githubusercontent.com/51381523/232735702-00b9176b-d828-4292-a1d3-f74181effaf8.png)

I'm also having issues with this. I ran `composer update`, `composer install` and the other suggested solutions.

**Comment by QazCetelic** [comment]
I was able to solve this issue by changing the page and thus discarding the cached page.

**Comment by gn306029** [comment]
> I don't know what was wrong exactly just cleared the space around added empty lines (above, bellow) the `@laravelPWA`
> 
> ```
> @laravelPWA
> ```
> 
> and it worked, I think the library could be improved using the [Manually registered component](https://laravel.com/docs/9.x/blade#manually-registering-components) and end up using something similar to this `<x-laravelpwa::meta/>` instead of the directive.

This work for me !
